test/e2e/storage/csi_objects.go

```diff
@@ -413,3 +420,75 @@ func deployGCEPDCSIDriver(
 	framework.ExpectNoError(err, "Failed to create DaemonSet: %v", nodeds.Name)
 
 }
+
+func createGCESecretsIfInCI(client clientset.Interface, config framework.VolumeTestConfig) {
+	var projectNumber string
+
+	gceCloud, err := framework.GetGCECloud()
+	framework.ExpectNoError(err, "Failed to get GCE Cloud")
+
+	_, err = strconv.ParseUint(gceCloud.ProjectID(), 10, 64)
+	if err == nil {
+		// Project ID is a number
+		projectNumber = gceCloud.ProjectID()
```

Comment:
which block is CI and which one is desktop?

Also in CI, does the test runner run in the same project where the cluster is created?

Reply:
you're right, I'm going to need to hack on this some more - the project here is the one where the test runner is run, not necessarily the one where the cluster is created.